jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/asserts/PaginationAssert.java

```diff
@@ -20,4 +25,91 @@ public PaginationAssert ended() {
+    public PaginationAssert notCircle() {
+        jdiAssert(element().isCircle() ? "circle" : "not circle", Matchers.is("not circle"));
+        return this;
+    }
+
+    @JDIAction("Assert that theme of '{name}' is light")
+    public PaginationAssert lightTheme() {
+        jdiAssert(element().isLightTheme(), Matchers.is(true));
+        return this;
+    }
+
+    @JDIAction("Assert that theme of '{name}' is dark")
+    public PaginationAssert darkTheme() {
+        jdiAssert(element().isDarkTheme(), Matchers.is(true));
+        return this;
+    }
+
+    @JDIAction("Assert that '{name}' aria-label has value '{0}'")
+    public PaginationAssert currentPageAriaLabel() {
+        jdiAssert(element().activeButton().attr("aria-label"),
+                Matchers.containsString(DEFAULT_CURRENT_PAGE_ARIA_LABEL));
+        return this;
+    }
+
+    @JDIAction("Assert that '{name}' previous-aria-label has value '{0}'")
+    public PaginationAssert previousAriaLabel() {
+        jdiAssert(element().getAriaLabelAttribute(element().leftNavigation()),
+                Matchers.containsString(DEFAULT_PREVIOUS_ARIA_LABEL));
+        return this;
+    }
+
+    @JDIAction("Assert that '{name}' next-aria-label has value '{0}'")
+    public PaginationAssert nextAriaLabel() {
+        jdiAssert(element().getAriaLabelAttribute(element().rightNavigation()),
+                Matchers.containsString(DEFAULT_NEXT_ARIA_LABEL));
+        return this;
+    }
+
+    @JDIAction("Assert that '{name}' next icon has value '{0}'")
+    public PaginationAssert nextIcon(String icon) {
+        jdiAssert(element().nextIcon().hasClass(icon),
+                Matchers.is(true));
+        return this;
+    }
+
+    @JDIAction("Assert that '{name}' next icon has value '{0}'")
+    public PaginationAssert previousIcon(String icon) {
+        jdiAssert(element().previousIcon().hasClass(icon),
+                Matchers.is(true));
+        return this;
+    }
+
+    @JDIAction("Assert that '{name}' page-aria-label has value '{0}'")
+    public PaginationAssert pageAriaLabel() {
+        element().list().foreach(button -> {
+            if (!button.hasClass(ITEM_CLASS_SELECTED)) {
+                jdiAssert(element().getAriaLabelAttribute(button),
+                        Matchers.containsString(DEFAULT_PAGE_ARIA_LABEL));
+            }
+        });
+
+        return this;
+    }
+    @JDIAction("Assert that '{name}' total visible has value '{0}'")
+    public PaginationAssert totalVisible(Integer totalVisible) {
+        jdiAssert(element().totalVisible(), Matchers.is(totalVisible));
+        return this;
+    }
+
+    @JDIAction("Assert that '{name}' color is '{0}'")
+    public PaginationAssert color(String color, Boolean isMaterialColor) {
+        if (isMaterialColor) {
+            jdiAssert(element().activeButton().hasClass(color), Matchers.is(true));
+        } else {
+            jdiAssert(element().activeButton().has().borderColor(color), Matchers.is(color));
```

Comment:
Can be border color differ from background color?

Reply:
<img width="952" alt="color" src="https://user-images.githubusercontent.com/74404787/196488704-8461d5a0-1fb9-49e7-9a9a-9ef2eac1e72b.png">
When I add color Props for pagination, the border-color and background-color will change, so I think need to be add both CSS color for that

Reply:
Hi @omega790, I fixed, please help me review on that